### PR TITLE
TASK: Refactor absolute namespaces in Media package

### PR DESCRIPTION
**Pull request by johannessteu**

### CODE REVIEW

File: TYPO3.Media/Classes/TYPO3/Media/TypeConverter/AssetInterfaceConverter.php [hunk 1]
```diff
@@ -131,9 +145,9 @@ public function getTypeOfChildProperty($targetType, $propertyName, PropertyMappi
     {
         switch ($propertyName) {
             case 'resource':
-                return 'TYPO3\Flow\Resource\Resource';
+                return Resource::class;
```
Comment (kitsunet) [inline TYPO3.Media/Classes/TYPO3/Media/TypeConverter/AssetInterfaceConverter.php]: PersistentResource

File: TYPO3.Media/Classes/TYPO3/Media/TypeConverter/AssetInterfaceConverter.php [hunk 2]
```diff
@@ -12,11 +12,25 @@
...
+use TYPO3\Flow\Persistence\PersistenceManagerInterface;
+use TYPO3\Flow\Property\Exception\InvalidDataTypeException;
+use TYPO3\Flow\Property\Exception\InvalidPropertyMappingConfigurationException;
+use TYPO3\Flow\Property\Exception\InvalidSourceException;
+use TYPO3\Flow\Property\Exception\InvalidTargetException;
+use TYPO3\Flow\Property\Exception\TargetNotFoundException;
 use TYPO3\Flow\Property\PropertyMappingConfigurationInterface;
+use TYPO3\Flow\Property\TypeConverter\ObjectConverter;
 use TYPO3\Flow\Property\TypeConverter\PersistentObjectConverter;
 use TYPO3\Flow\Resource\Resource;
```
Comment (kitsunet) [inline TYPO3.Media/Classes/TYPO3/Media/TypeConverter/AssetInterfaceConverter.php]: PersistentResource

**Comment by johannessteu** [comment]
Fixed

**Comment by robertlemke** [comment]
There were 3 occurrences of "Resource" which haven't been changed into "PersistentResource" in `AssetInterfaceConverter`. Will fix that in a follow-up PR.

**Pull request merged**